ext/php5/php5/engine_hooks.c

```diff
@@ -311,6 +311,10 @@ static int dd_exit_handler(zend_execute_data *execute_data TSRMLS_DC) {
 }
 
 static ddtrace_dispatch_t *dd_lookup_dispatch_from_fbc(zend_function *fbc TSRMLS_DC) {
+    if (!PG(modules_activated)) {
```

Comment:
Since this is a hot path, do you think it is worth wrapping this in an `UNEXPECTED`?

Reply:
Not in isolation, for there to be significance, we would have to break down into more branches with the other conditions, and determine which ones are expected/not expected ...

tl;dr doesn't seem worth it.